docs/components-release.rst

```diff
@@ -0,0 +1,16 @@
+Components release
+==================
+
+The following sections summarize the release process of the various components of the software stack.
+Note this is not an exhaustive list of all tools release process.
+
+
+.. toctree::
+    :maxdepth: 1
+
+    omero-release-process
+    omero-web-release-process
+    gradle-development
+    java-development
+    cpp-development
```

Comment:
This might possible be marked as deprecated (all repositories should now be archived)